src/services/types.ts

```diff
@@ -16,6 +16,7 @@ export enum CustomEvents {
     CAPTURE_TEST = 'Capture_Test',
     COLLECTION_CREATE = 'Collection_Create',
     CONNECTOR_VERSION_UNSUPPORTED = 'Connector_Version:Unsupported',
+    DATA_PLANE_SELECTOR_DEFAULTED = 'Data_Pane_Selector:Unexpectedly_Defaulted',
```

Comment:
Sorry - this isn't really the pattern anymore.

You can just have the event be `DATA_PLANE_SELECTOR` and then set properties in the data that is being passed to LR. I noticed that it makes updating queries and segments a lot easier that way.